determined-ai/determined
# chore(type): stricter tsconfig

## Description
[WEB-1773]




## Test Plan
- Check if there's no typescript errors
- Check if all changes are valid
- Check if all workflows work as expected such as login, logout, create something, edit something, fetch something etc




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1773]: https://hpe-aiatscale.atlassian.net/browse/WEB-1773?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
